apps/dashboard/components/transaction-details/transaction-item.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
```

Comment:
what made you choose mui? did you consider other library? we're not using it for other front end repo (but maybe we could/should - would like to understand your take here)

Reply:
MUI is one of the most popular React component libraries and has a large community, so I thought it would be more stable with Next.js and would be updated frequently due to its many contributors. It is also used by tech giants, as seen on https://mui.com/.

However, we can use something else if you have any other preference.